python/pyspark/ml/classification.py

```diff
@@ -857,6 +871,12 @@ def featureImportances(self):
         """
         return self._call_java("featureImportances")
 
+    @property
+    @since("2.0.0")
+    def trees(self):
```

Comment:
@yanboliang this one should return classification models (RFClassifier is the only one which does, both GBT models return regression models).